JS-Algorithms/challenges.js

```diff
@@ -38,10 +46,15 @@ Example:
 Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "Germany", "Norway"]"
 ***** */
 
-const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
+const COUNTRY_NAMES = ['Germany', 'Norway', 'Island', 'Japan', 'Israel'];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let copyArray = [...COUNTRY_NAMES];
+  if (index >= COUNTRY_NAMES.length) {
```

Comment:
same comment regarding spaces, and for readibility we could add blank spaces before anf after an if statement